lib/xlsxtream/io/zip_tricks_io.rb

```diff
@@ -0,0 +1,26 @@
+require "zip_tricks"
+
+module Xlsxtream
+  module IO
+    class ZipTricksIO
```

Comment:
Please remove "IO" suffix from the class and file name, it is redundant because it's already in the `IO`  module.